code/game/gamemodes/clock_cult/clock_structures.dm

```diff
@@ -462,9 +462,10 @@
 		return ..()
 
 /obj/structure/clockwork/mending_motor/proc/toggle(mob/living/user)
-	if(!user || !is_servant_of_ratvar(user))
-		return 0
-	user.visible_message("<span class='notice'>[user] [active ? "en" : "dis"]ables [src].</span>", "<span class='brass'>You [active ? "en" : "dis"]able [src].</span>")
+	if(user)
+		if(!is_servant_of_ratvar(user))
+			return 0
+		user.visible_message("<span class='notice'>[user] [active ? "en" : "dis"]ables [src].</span>", "<span class='brass'>You [active ? "en" : "dis"]able [src].</span>")
```

Comment:
you need to swap the `[active ? "en" : "dis"]`es to `[active ? "dis" : "en"]` because this message is before the active = !active toggle